jni/nnstreamer.mk

```diff
@@ -109,6 +109,12 @@ NNSTREAMER_FILTER_CAFFE2_SRCS := \
 NNSTREAMER_FILTER_SNPE_SRCS := \
     $(NNSTREAMER_EXT_HOME)/tensor_filter/tensor_filter_snpe.cc
 
+# Check whether the version of SNPE SDK is old (v1, rather than v2)
+ifeq ($(shell test -d ${SNPE_ROOT}/include/zdl; echo $$?),0)
+NNSTREAMER_FILTER_SNPE_SRCS := \
+    $(NNSTREAMER_EXT_HOME)/tensor_filter/tensor_filter_snpe_v1.cc
+endif
```

Comment:
Isn't this too dangerous assumption?

I'd recommend:

- build snpe(v2) if v2 is available
- build snpe(v1) if v1 is available AND v2 is not available.

The current logic is

- build snpe(v2) if v2 is available AND v1 is NOT available
- build snpe(v1) if v1 is available AND.

Reply:
I assume $SNPE_ROOT is unique and a single SNPE SDK does not have both header directories for V1 and V2. I think there is no possibility to both v1 and v2 available.

Reply:
ok.. if ```!exist(zdl)``` means !v2, let's proceed.